saa_victoria_subtheme.react_behaviors.yml

```diff
@@ -23,6 +23,14 @@ card_variant_info:  # part of the key after the :
       options:
         su-card-wrapper--border-none: Minimal (no border)
         su-card-wrapper--border-regular: Border and Drop Shadow
+    card_list_style:
+      type: select
+      title: 'Card List Style'
+      description: 'Select the style for list format. First paragraph tag will appear regular and all following paragraphs will be styled as a list.'
+      options:
+        su-card-wrapper--no-style: No List Style
```

Comment:
Katria had made a note to create 2 boolean fields. I believe that we would only want the editor to be able to choose one of the two styles so I decided to create a select list instead. 🤔 

I created the `su-card-wrapper--no-style : No List Style` as the default list style in the event that the editor would like to unselect the list style.

Reply:
Oh good. I like the no list style for the default.